pkg/file/management_test.go

```diff
@@ -1,27 +1,73 @@
 package file
 
 import (
+	"flag"
+	"io"
+	"os"
+	"path/filepath"
 	"testing"
 
 	"github.com/stretchr/testify/require"
 )
 
+var (
+	update = flag.Bool("update", false, "update the golden files of this test")
+)
+
+func TestMain(m *testing.M) {
+	flag.Parse()
+	os.Exit(m.Run())
+}
+
+// Note: to update the goldens, run the tests with `-update`:
+//
+// go test ./pkg/file/... -update
```

Comment:
No need to change as it's nice to have it like that too, but note that flags defined in tests can be used without a `TestMain` by passing `-test.update` (i.e. prepending `test.` to the flag).

Reply:
TIL, thank you! you mentioned that's not necessary but I'm happy to change it, just lmk

Reply:
I don't think it's important enough, it was just to mention the test prefix as it's pretty hard to discover (Go tooling is great but a lot is hidden in `go help` which is also a bit awkward in syntax!).